core/arch/arm/mm/core_mmu.c

```diff
@@ -967,7 +967,9 @@ static void check_pa_matches_va(void *va, paddr_t pa)
 			panic("issue in linear address space");
 		return;
 	}
-	if (v >= (CFG_TEE_LOAD_ADDR & ~CORE_MMU_PGDIR_MASK) &&
+	/* wait pager is initialize before looking into pager data */
+	if (cpu_mmu_enabled() &&
```

Comment:
This isn't a guarantee that the pager is initialized.
Testing `tee_pager_tbl_info.table` is better, for our purpose probably even OK.

Do you know where and what we're looking up to cause the error?